requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
 sklearn
-topopy
+nglpy>=1.0.6,<2.0
+topopy>=1.0,<2.0
```

Comment:
I am explicitly pinning versions here, to coincide with semantic versioning. If I bump the major topopy version again, it would be a breaking API change, otherwise, minor version and patch version changes should be grabbed.